tools/common/Frameworks.cs

```diff
@@ -341,7 +342,7 @@ public static Frameworks GetiOSFrameworks (Application app)
 	}
 
 	static Frameworks watch_frameworks;
-	public static Frameworks GetwatchOSFrameworks (Application app)
+	public static Frameworks GetwatchOSFrameworks (bool is_simulator_build)
```

Comment:
Question, not that is a big deal, why not camelCase?

Reply:
Because of consistency and branding!! 

- iOS
- watchOS
- macOS
- tvOS

Reply:
> Question, not that is a big deal, why not camelCase?

Because I can never remember which casing to use for parameters.

Maybe I'll remember for next time now :smile:

Reply:
if speaking of parameters then it should be camelCase as per FXDG
for the method name then @dalexsoto comment is right - looks weird in any case :)